### PR TITLE
fix(analytics): preserve whitespace for exports

### PR DESCRIPTION
**Pull request by Yelinz**
## Description

exports do not include white space currently

This should fix that

### CODE REVIEW

File: packages/analytics/addon/components/ca-report-preview.hbs [hunk 1]
```diff
@@ -30,7 +30,10 @@
         {{#each this.data.value.fields as |row|}}
           <tr>
             {{#each row as |entry|}}
-              <td data-t={{this.getXLSXType entry.value}}>
+              <td
+                data-t={{this.getXLSXType entry.value}}
+                data-v={{this.cleanValue entry.value}}
```
Comment (derrabauke) [inline packages/analytics/addon/components/ca-report-preview.hbs:35]: Where are those attributes consumed? Couldn't find any docs. :man_shrugging:

Comment (Yelinz) [reply packages/analytics/addon/components/ca-report-preview.hbs:35]: Used in the excel export of the analytics table. Definition is here https://docs.sheetjs.com/docs/csf/cell

Comment (derrabauke) [reply packages/analytics/addon/components/ca-report-preview.hbs:35]: Thanks! After skimming over the dox I've found this one to be even more appropriate: https://docs.sheetjs.com/docs/api/utilities/html#implementation-details

**Review by derrabauke — APPROVED** [review]

**Pull request merged**